src/uopool/services/sanity_check.rs

```diff
@@ -296,17 +240,9 @@ where
         user_operation: &UserOperation,
         entry_point: &Address,
     ) -> Result<(), BadUserOperationError<M>> {
-        self.sanity_check_results.write().insert(
-            user_operation.hash(entry_point, &self.chain_id),
-            Default::default(),
-        );
-
         // Either the sender is an existing contract, or the initCode is not empty (but not both)
         self.sender_or_init_code(user_operation).await?;
 
-        // If initCode is not empty, parse its first 20 bytes as a factory address. Record whether the factory is staked, in case the later simulation indicates that it needs to be. If the factory accesses global state, it must be staked - see reputation, throttling and banning section for details.
-        self.verify_factory(user_operation, entry_point).await?;
```

Comment:
Curious why this is removed.

Reply:
The only thing that was verified here is if the factory is staked. This is now available here: https://github.com/Vid201/aa-bundler/blob/feat/uo_trace/src/uopool/services/simulation.rs#L98, so no need to check that here also.